app/controllers/orgs/rosters_controller/lti_dependency.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Orgs
+  class RostersController
+    class LtiImportError < StandardError; end
```

Comment:
Love this structure ❤️

Reply:
I'm so happy we both discovered it. For another time, but GClassroom's integration with RostersController could probably easily be refactored into a similar dependency.

Reply:
Yes! Once we get the UI changes in.